packages/amp/src/index.tsx

```diff
@@ -101,7 +101,7 @@ const amp: AMP = {
               // Else capture all the script and link with as="script" tags.
               else if (/<?script/g.test(tag)) {
                 // And if they contain the `amp-` specific markers keep them.
-                if (/<?script.+?amp-/g.test(tag)) {
+                if (/<?script.+?="amp-/g.test(tag)) {
```

Comment:
Just to be sure - this would allow only those scripts with `custom-element="amp-..."`, right?

What would happen with the main AMP library?

```html
<script async src="https://cdn.ampproject.org/v0.js"></script>
```

Reply:
The main script is part of the template, as far as I remember, not part of the dynamic head scripts.

Here: https://github.com/frontity/frontity/blob/dev/packages/amp/src/template.ts#L19